app/.storybook/i18next.js

```diff
@@ -5,10 +5,13 @@ import LanguageDetector from "i18next-browser-languagedetector";
 import Backend from "i18next-http-backend";
 import { initReactI18next } from "react-i18next";
 
+import i18nConfig from "../next-i18next.config";
+
+const { i18n: i18nOptions, ...i18nextOptions } = i18nConfig;
```

Comment:
Not too familiar with some of this syntax, so just want to make sure I understand what's happening: the import statement imports the `module.exports` from `next-i18next.config.js` as `i18nConfig`, then the `i18n` part is saved to `i18nOptions`, and the rest is saved to `i18nextOptions`. That's what I think I'm seeing based on [this](https://www.typescriptlang.org/docs/handbook/variable-declarations.html#object-destructuring), is that right?

Reply:
yea this was a little funky. The two pieces here are JS syntax:

- [Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
  - `...i18nextOptions` is a "[rest property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#rest_property)"
  - `i18n: i18nOptions` stores `i18nConfig.i18n` to a const named `i18nOptions`

I just simplified some of this by renaming the `i18n` import so we don't have to rename `i18n` to `i18nOptions` here. https://github.com/navapbc/template-application-nextjs/pull/59/commits/ad7fbb37cbd68e308b8821177dc638402bb1eb25

Reply:
Gotcha, thanks for spelling that out, the rename helps too!